charts/multicluster-scheduler/templates/cr.yaml

```diff
@@ -236,6 +239,17 @@ rules:
   - apiGroups: [""]
     resources: ["pods"]
     verbs: ["list"]
+  - apiGroups: [""]
+    resources: ["pods/finalizers"]
+    verbs:
```

Comment:
If I just use the update verb, though the jobs run to the completion successfully, somehow the pods remain there with the completed status. If I try to delete the pods, it will be stuck with pending. 
```
apiVersion: batch/v1
kind: Job
metadata:
  name: hf0-job
spec:
  template:
    metadata:
      annotations:
        multicluster.admiralty.io/elect: ""
    spec:
      containers:
      - name: c
        image: busybox
        command: ["sh", "-c", "echo Processing item 123 && sleep 5"]
        resources:
          requests:
            cpu: 100m
      restartPolicy: Never
```
This is my job yaml file.

The strange behavior is shown below:
```
# oc delete jobs --all
No resources found

# oc get pods
NAME                   READY   STATUS      RESTARTS   AGE
hf0-job-pc9wp-ffbqf    0/1     Completed   0          2m24s

[root@hf-eu-login admiralty]# oc delete pod --all
pod "hf0-job-pc9wp-ffbqf" deleted

[root@hf-eu-login admiralty]# oc get pods
NAME                   READY   STATUS    RESTARTS   AGE
hf0-job-pc9wp-ffbqf    0/1     Pending   0          2s
```
Maybe you have the answer to this. After numerous tries, I think keeping all the verbs there provide the best behavior (in terms of cleaning up).

Reply:
This looks to me like you're trying to delete delegate pods in the target cluster directly. Why would you do that? If you do that, instead of deleting corresponding proxy pods in the source cluster, delegate pods are recreated by their pod chaperons. Notice the age of the pod after your last `oc get pods`.

Reply:
No. I would always delete the jobs from the source cluster.

Reply:
If I remember correctly, the source is one of the target clusters as well. Anyway,  as mentioned above, after adjusting SCC, keeping just the update verb works fine here.  FIles are updated.